### DISCUSSION TITLE
Sonoff NS Panel nspanel.be throws errors

### DISCUSSION THREAD

**Discussion by kapollo** [arendst/Tasmota#17306] [Q&A] [answered]
Hi have anyone recently tried to use Tasmota 12 with NSPanel? I seem to be unable to figure out one error I'm getting with nspanel.be
I'm using the most recent berry file provided [here ](https://github.com/blakadder/nspanel/blob/main/nspanel.be) by blakeadder
The log from uart is
```
<snip snip snip snip>
00:00:00.002 HDW: ESP32-D0WD-V3
00:00:00.043 UFS: FlashFS mounted with 288 kB free
00:00:00.082 CFG: Loaded from File, Count 12
00:00:00.092 QPC: Count 1
00:00:00.216 BRY: Berry initialized, RAM used=4041 bytes
00:00:00.234 Project tasmota - Tasmota Version 12.2.0.6(nspanel)-2_0_5_3(2022-12-07T19:49:51)
<snip snip snip snip>
20:13:36.245 BRY: Exception> 'index_error' - list index out of range
20:13:36.247 stack traceback:
20:13:36.247    <native>: in native function
20:13:36.248    <unknown source>: in function set_power
20:13:36.260    <unknown source>: in function screeninit
20:13:36.261    <unknown source>: in function <lambda>
20:13:36.272    <unknown source>: in function try_rule
20:13:36.273    <unknown source>: in function exec_rules
20:13:36.274    <unknown source>: in function event`
<snip snip snip snip>
```

Any ideas?

**Comment by sfromis** [comment, marked as answer]
Do you have the correct config template active? The [Berry code](https://github.com/blakadder/nspanel/blob/main/nspanel.be#L160-L173) looks like the index out of range may happen in case of not having two power channels available to Tasmota.

**Reply by kapollo** [reply]
Thanks @sfromis 
I feel like looking at the mirror saying rtfm ...
It works fine now.
